### PR TITLE
Use the same CDATA color for xml literals as for doc comments

### PR DESCRIPTION
**Pull request by JoeRobich**
Fixes https://github.com/dotnet/roslyn/issues/44406 and https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1139744

Before:
image

After:
image

/CC: @Cosifne

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Can you show a before/after?

**Comment by JoeRobich** [comment]
> Can you show a before/after?

@CyrusNajmabadi Added examples to the description.

File: src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Classification
 {
     internal sealed class ClassificationTypeFormatDefinitions
     {
-        #region Preprocessor Text 
+        #region Preprocessor Text
```
Comment (Youssef1313) [inline src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs:21]: Is IDE0055 disabled? It should have caught the extra space if enabled

Comment (sharwell) [reply src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs:21]: IDE0055 is enabled. The formatter currently fails to remove certain end-of-line whitespaces.

**Pull request merged**